hack/build-test-bin.sh

```diff
@@ -8,13 +8,15 @@ if ! which go; then
 fi
 
 GOPATH="${GOPATH:-~/go}"
+GOFLAGS="${GOFLAGS:-"-mod=vendor"}"
+
 export PATH=$PATH:$GOPATH/bin
 
 if ! which gingko; then
```

Comment:
now that we use the tools package can you change this to build it from the vendor folder?

Reply:
it works by having GOFLAGS=-mod=vendor.
The issue is that in the openshift base image GOFLAGS is already set like that, hence line 12.
With that variable, all build happen using the fendor folder.

Reply:
great thanks!